README.md

```diff
@@ -1,2 +1,44 @@
-# ActiveLearningAPI
-API for active learning
+# Active Learning API
+
+
+We use [FastAPI](https://fastapi.tiangolo.com/).
```

Comment:
This is not the most important aspect of this code-base, so shouldn't be the first line.

Instead, I'd start with a description of what problem this code-base is trying to solve, and how it goes about it.